apps/framework-cli/src/cli.rs

```diff
@@ -151,12 +120,32 @@ pub async fn top_command_handler(
             location,
             template,
             no_fail_already_exists,
+            from_remote,
+            language,
+            // database,
```

Comment:
Maybe can you split the body of this in smaller functions? that looks a lot of cases that are nested which males it harder to review